### ISSUE TITLE
Bounties: for supporting us on social media, chats, etc.

### ISSUE THREAD

**Issue by Ivshti** [AdExNetwork/adex-staking#74]
## Summary

This was previously discussed in [AIP 48](https://github.com/AdExNetwork/aips/issues/48) as "fun pool" but it better translates to a new bounties feature.

Add a new "Bounties" page where we reward users with ADX for joining our soical media or communities

## Rewards

Reward users for:

* Early supporter: held ADX in 2017
* Community member: is in Telegram/Discord
  * we can make this interesting by having a "daily secret" or "daily password" and users have to enter at least 2-3 to get the reward, making it harder to sybil too
* Retweets our tweets - you get ADX if your account has been created before a certaint time, has a certain number of followers, and you don't unretweet
* Referring users to the platform - referral/affiliate program - get ADX minted for every DAI spent/earned by users you referred, up to a certain limit

We can gamify it by adding badges too, similarly to how the platform does it when you're a new user.

## Implementation

Use an OUTPACE payment channel.

Checking stuff like whether you're an early supporter is easy: just use a preset list of all 2017 holder addrs, and if you're one of them, you can claim. No need to check for signed msgs or anything like that since the reward would go directly to the addr that requested it.

## Challenges

* How to prove the user is real (anti-sybil)?
* How to prove the user has joined?
* How to ensure the user is engaged?

Basically it's all very gamable so we have to figure it out as much as possible

**Comment by dmngdsm** [comment]
Let's include Telegram quizzes too. Can be daily AdEx Trivia for small rewards

**Comment by Ivshti** [comment]
Further ideas:
* TG: AMA and all eligible questions will participate in a raffle
* TG: Draw contest idea to get more people to staking, very simply 1.stake at least 100 adx to win each 100 is an additional chance to win amount can be changed rewards let say 10k pool 5k,2k,1k 20x100 to participate they have to join tg channel or discord channel and paste their transaction id

**Comment by Ivshti** [comment]
@vanina-iv @dmngdsm 

I am going to close this for now. However, by no means should this be interpreted as deprioritizing growing TG. In fact, the contrary. I just think are far more efficient in terms of results and ease of execution, and we should do another one, along with AMAs
